docs/content/modules/rdg/introduction/Overview.md

```diff
@@ -0,0 +1,339 @@
+
+!image media/rdg/fig_1d_aefv_square_wave.png width=50% padding-right=20px float=right caption=Instantaneous rDG solution for simulating a right-going square-shaped wave in 1D.
+
+
+!image media/rdg/anim_1d_aefv_square_wave.gif width=50% padding-right=20px float=right caption=Time evolution of rDG solution for simulating a right-going square-shaped wave in 1D.
+
+###Input File
+
+The content in some of the input keyword blocks is described in detail for clarity.
+
+####GlobalParams
+
+!input modules/rdg/tests/advection_1d/1d_aefv_square_wave_minmod.i block=GlobalParams
+
+Notes:
+
+  1. It is mandatory to declare `order = CONSTANT` and `family = MONOMIAL`, which specifies the piecewise cell-average solution variable for the cell-centered finite volume method.
+  2. `u = u` means a name `u` is assigned to the piecewise cell-average solution variable `u`.
+  3. It is mandatory to give some user object names like `slope_reconstruction = rslope` and `slope_limiting = lslope` for *slope reconstruction* and *slope limiting*.
+  4. If an explicit time integration method is used, it is convenient to declare `implicit = false` here, so that Jacobian matrices will not be computed.
+
+####Mesh
+
+!input modules/rdg/tests/advection_1d/1d_aefv_square_wave_minmod.i block=Mesh
+
+Notes:
+
+  1. A one-dimensional computational domain ranging between $x=0$ and $x=1$ is defined, with 100 elements equally distributed in the domain.
+
+####Functions
+
+!input modules/rdg/tests/advection_1d/1d_aefv_square_wave_minmod.i block=Functions
+
+Notes:
+
+  1. In this case, a piecewise constant function is used to specify the initial condition of the square-shaped wave profile.
+
+####UserObjects
+
+!input modules/rdg/tests/advection_1d/1d_aefv_square_wave_minmod.i block=UserObjects
+
+Notes:
+
+  1. The prefix `AEFV` represents "Advection Equation Finite Volume" --- an abbreviation to uniquely name these classes.
+  2. `AEFVSlopeReconstructionOneD` does not do any work, but has to be in place for code consistency. In 1D the slope reconstruction and limiting can be accomplished in one user object.
+  3. `AEFVSlopeLimitingOneD` calculates the limited slope for each element in 1D.
+  4. `AEFVUpwindInternalSideFlux` calculates the internal side flux using a simple upwind scheme.
+  5. `AEFVFreeOutflowBoundaryFlux` calculates the boundary side flux using a free outflow BC.
+
+####Variables
+
+!input modules/rdg/tests/advection_1d/1d_aefv_square_wave_minmod.i block=Variables
+
+Notes:
+
+  1. Declare the nonlinear variable as `u`. The type and family of the variable have been declared in `GlobalParams` for convenience.
+
+####Kernels
+
+!input modules/rdg/tests/advection_1d/1d_aefv_square_wave_minmod.i block=Kernels
+
+Notes:
+
+  1. Always set `implicit = true` for time derivative kernels.
+  2. In FVMs, there is no volumetric integration for flux terms.
+  3. Other possible kernels in this block are source terms, which though are not needed here.
```

Comment:
Other possible kernels in this block would include source terms. In this example, we do not have any.